packages/x-tailwindcss/src/x-tailwind-plugin/components/button-group/separator.ts

```diff
@@ -0,0 +1,19 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the styles for the `button-group-separator`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function buttonGroupSeparator(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    separator: {
+      borderLeft: '1px solid currentColor',
+      margin: `${theme('spacing.8')} 0`,
```

Comment:
According to design, it should be `4px` of `paddingY` instead of `8px`, isn't it?
```suggestion
      margin: `${theme('spacing.4')} 0`,
```

Reply:
Sorry about that, I forgot to tell Alicia to update the designs to apply this change and the default color one 🤦 
It is 8px 👌